### ISSUE TITLE
how to handle license plate id = -1 ? 

### ISSUE THREAD

**Issue by xuexingyu24** [srihari-humbarwadi/DeepLabV3_Plus-Tensorflow2.0#15]
Hi srihari 

Thanks for your excellent job!

Since the trained model has 34 channel output. The model could predict all items including license plate which has id = -1. I am wondering how the tensorflow cross entropy loss handles the label with negative value ?

**Comment by srihari-humbarwadi** [comment]
Yeah, you are right. I am planning to retrain the model only on classes which are used in final evaluation. 
And as far as I know, there aren't any pixels with -1 in the dataset. But I will have to cross check this. Thanks for bringing this up!

**Comment by Alokksinha00** [comment]
Hi Srihari,
1. How similar your uploaded code is with original paper authors LC et all. As i find quite different than yours (yours is more readable actually).
Or is it just TF version difference (I am slightly new to tensorflow).
2. Can I just take your code and run locally (w/o gpu)?
3. I also see you haven't included xception in your model?

**Comment by srihari-humbarwadi** [comment]
1. The ASPP module is implemented as described in the paper. There are multiple possibilities for the feature extraction backbone. The paper has two variants, ResNet101 and a modified Xception backbone, but for sake of inference speed and simplicity I decided to go ahead with Resnet50 with a fixed output stride. I am planning to add a few more backbones in the coming weeks.
2. Yes you can run this code on CPU locally. I wouldn't suggest training on it though!
